.github/workflows/lint.yml

```diff
@@ -36,6 +36,18 @@ jobs:
           node-version: '12.x'
       - run: npm install -g markdownlint-cli@0.25.0
       - run: markdownlint '**/*.md'
+  spelling:
```

Comment:
This file is more like a linter collection, we may have a `checker` file as well?

Reply:
We already put the `spelling` in a separate file. There is no need to put it into lint.